client/src/components/AuditBoard/BallotAudit.tsx

```diff
@@ -1,7 +1,6 @@
-import React from 'react'
-import { Formik, FormikProps, Field } from 'formik'
+import React, { useState } from 'react'
```

Comment:
Before proceeding with my state cleanup, I decided to remove `Formik` from this form to simplify things 🧹

I recommend reviewing this commit using [`?w=1`](https://github.com/votingworks/arlo/pull/1583/commits/6a556ec5ccc9822f324eaacdca86f36d66290eed?w=1)

Reply:
Hurrah! Formik is a pain in the butt. I started using react-hook-form instead and have found that to be a lot nicer to use.